TramsDataApi.Test/Integration/EstablishmentsIntegrationTests.cs

```diff
@@ -225,11 +240,12 @@ public async Task CanSearchEstablishmentsByPartialUkprn()
             result.Should().BeEquivalentTo(expected);
         }
 
-        private EstablishmentResponse AddTestData(int urn)
+        private EstablishmentResponse AddTestData(int urn, string? region)
```

Comment:
Small thing: could make this an optional parameter.

Reply:
Agree, have changed.